core/frame/data_frame.go

```diff
@@ -70,6 +70,16 @@ func (d *DataFrame) SourceID() string {
 	return d.metaFrame.SourceID()
 }
 
+// SetDispatch set dispatch mode
+func (d *DataFrame) SetDispatch(mode Dispatch) {
```

Comment:
in DataFrame, `SetBroadcast` might be better

Reply:
I'm not sure we have any other model.